pkg/cloud/aws/commands/run.go

```diff
@@ -124,13 +125,58 @@ func Run(ctx context.Context, opt flag.Options) error {
 			return fmt.Errorf("aws scan error: %w", err)
 		}
 	}
-	r := report.New(cloud.ProviderAWS, opt.Account, opt.Region, results.GetFailed(), opt.Services)
 
 	log.Logger.Debug("Writing report to output...")
+	if len(opt.Compliance) > 0 && opt.ReportFormat == "summary" {
```

Comment:
The compliance framework also provide data when `opt.ReportFormat == "all"` not just `summary` and also support json and table view.

so in total you can produce four types of output:
- table with report == summary
- table with report == all
- json with report == summary
- json with report == all

Reply:
`--format=json --report=summary`: https://github.com/aquasecurity/trivy/pull/3130/files#diff-cccb8f0a54933d9201a68010d9f686e9f61bd16677fb5e8d21c9c8b3ff6bf2ddR139-R161

`--format=json --report=all`: 
```json
{
  "ArtifactName": "<redacted>",
  "ArtifactType": "aws_account",
  "Metadata": {
    "ImageConfig": {
      "architecture": "",
      "created": "0001-01-01T00:00:00Z",
      "os": "",
      "rootfs": {
        "type": "",
        "diff_ids": null
      },
      "config": {}
    }
  },
  "Results": [
    {
      "Target": "arn:aws:iam::<redacted>:user/AzureADRoleManager",
      "Class": "config",
      "Type": "cloud",
      "MisconfSummary": {
        "Successes": 0,
        "Failures": 4,
        "Exceptions": 0
      },
      "Misconfigurations": [
        {
          "Type": "AWS",
          "ID": "AVD-AWS-0144",
          "Title": "Credentials which are no longer used should be disabled.",
          "Description": "CIS recommends that you remove or deactivate all credentials that have been unused in 90 days or more. Disabling or removing unnecessary credentials reduces the window of opportunity for credentials associated with a compromised or abandoned account to be used.",
          "Message": "User access key '<redacted>' has not been used in \u003e90 days",
          "Resolution": "Disable credentials which are no longer used.",
          "Severity": "MEDIUM",
          "PrimaryURL": "https://avd.aquasec.com/misconfig/avd-aws-0144",
          "References": [
            "https://avd.aquasec.com/misconfig/avd-aws-0144"
          ],
          "Status": "FAIL",
          "Layer": {},
          "CauseMetadata": {
            "Resource": "arn:aws:iam::<redacted>:user/AzureADRoleManager",
            "Provider": "aws",
            "Service": "iam",
            "Code": {
              "Lines": null
            }
          }
        },
      <snip>..
    }
  ]
}
```

`--format=table --report=summary`: https://github.com/aquasecurity/trivy/pull/3130/files#diff-fe969e5428e7e462874a674a3116c74787735b8a6d25f113cd3c06653433eb38

`--format=table --report=all`: https://github.com/aquasecurity/trivy/pull/3130/files#diff-7f47b6c95941e5b66cd6ccd4350bce2bf414042f8ae7d25b8157c298e96dc282 (this combination today doesn't actually return a table in trivy k8s FYI).


Please let me know if I've missed something.